code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -719,11 +718,9 @@ \subsubsection{Data Definitions}
               \item{$ϕ$ is the orientation (rad)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & \hyperref[assumpOT]{A: objectTy}
+Notes & All bodies are assumed to be rigid (from \hyperref[assumpOT]{A: objectTy}) and two-dimensional (from \hyperref[assumpOD]{A: objectDimension}).
```

Comment:
We do not want to remove the 2D assumption here, since we really only focus on one angle.  (The notation suggests that we have a more generalized notion of rotation, but that isn't really true.)  We can remove the reference to damping though.